### ISSUE TITLE
Bug: [no-unused-vars] Error message says "args" for "caught errors"

### ISSUE THREAD

**Issue by kirkwaiblinger** [eslint/eslint#18606]
### Environment

[playground](https://eslint.org/play/#eyJ0ZXh0IjoidHJ5IHtcbiAgdGhyb3cgbmV3IEVycm9yKCk7XG59IGNhdGNoIChfKSB7IFxufSIsIm9wdGlvbnMiOnsicnVsZXMiOnsibm8tdW51c2VkLXZhcnMiOlsid2FybiIseyJjYXVnaHRFcnJvcnNJZ25vcmVQYXR0ZXJuIjoiZm9vIn1dfSwibGFuZ3VhZ2VPcHRpb25zIjp7InNvdXJjZVR5cGUiOiJtb2R1bGUiLCJwYXJzZXJPcHRpb25zIjp7ImVjbWFGZWF0dXJlcyI6e319fX19)

### What parser are you using?

Default (Espree)

### What did you do?

```
"no-unused-vars": ["warn", { "caughtErrorsIgnorePattern": "foo" }]
```

```js
try {
} catch (_) { 
}
```

### What did you expect to happen?

'_' is defined but never used. Allowed unused ***caught errors*** must match /foo/u.  

### What actually happened?

'_' is defined but never used. Allowed unused ***args*** must match /foo/u.

### Link to Minimal Reproducible Example

https://eslint.org/play/#eyJ0ZXh0IjoidHJ5IHtcbiAgdGhyb3cgbmV3IEVycm9yKCk7XG59IGNhdGNoIChfKSB7IFxufSIsIm9wdGlvbnMiOnsicnVsZXMiOnsibm8tdW51c2VkLXZhcnMiOlsid2FybiIseyJjYXVnaHRFcnJvcnNJZ25vcmVQYXR0ZXJuIjoiZm9vIn1dfSwibGFuZ3VhZ2VPcHRpb25zIjp7InNvdXJjZVR5cGUiOiJtb2R1bGUiLCJwYXJzZXJPcHRpb25zIjp7ImVjbWFGZWF0dXJlcyI6e319fX19

### Participation

- [X] I am willing to submit a pull request for this issue.

### Additional comments

Looks like this is just a copypasta mistake; see 

https://github.com/eslint/eslint/blob/97ce45bcdaf2320efd59bb7974e0c8e073aab672/lib/rules/no-unused-vars.js#L163-L185

Changing line 170 to `variableDescription = "caught errors";` would ostensibly do the trick

**Comment by nzakas** [comment]
Confirmed as a bug. Please feel free to open a PR. :)

**Comment by kirkwaiblinger** [comment]
Cool! Will do so soon! 🙂
